contracts/BatchExchange.sol

```diff
@@ -233,7 +233,7 @@ contract BatchExchange is EpochTokenLocker {
       * @param sellAmounts maximum amounts of sell token to be exchanged in new orders
       * @return an array of indices in which `msg.sender`'s new orders are included
       *
-      * Emits {OrderCancelation} events for all cancelled orders and {OrderPlacement} events with all relevant new order details.
+      * Emits {OrderCancellation} events for all cancelled orders and {OrderPlacement} events with relevant new order details.
```

Comment:
Personally, I always go for Canadian/British spelling, so I prefer "cancelled". If we want to use American spelling, then we should change all instances of "cancelled" to "canceled" as well. Note that this change should be in a separate PR if we decide to do the change.

Reply:
I also prefer this in some situations. However I prefer to use a Z in words like realization and stuff like that.

Reply:
Yes... I agree separate PR... I wish I had made more specific commits.